locale/en/docs/guides/debugging-getting-started.md

```diff
@@ -241,7 +241,7 @@ couple popular ones are listed below.
 
 The V8 Debugging Protocol is no longer maintained or documented.
 
-#### [Built-in Debugger](https://nodejs.org/dist/latest-v6.x/docs/api/debugger.html)
+#### [Built-in Debugger](https://nodejs.org/docs/latest/api/debugger.html)
 
 Start `node debug script_name.js` to start your script under Node's builtin
```

Comment:
So if you're here, there are two other changes the should be done:
1. Remove the next section (`#### node-inspector`) [REF](https://github.com/node-inspector/node-inspector#overview)
2. replace `debug` with `inspect`
```suggestion
Run `node inspect script_name.js` to start your script under node's builtin
```

And similar in L248